src/sign-eip7702-authorization.test.ts

```diff
@@ -88,17 +88,45 @@ describe('signAuthorization', () => {
       ).toThrow('Missing chainId parameter');
     });
 
-    it('should throw if contractAddress is null', () => {
+    it('should throw if chainId is not a number', () => {
       expect(() =>
         signEIP7702Authorization({
           privateKey: TEST_PRIVATE_KEY,
           authorization: [
-            TEST_AUTHORIZATION[0],
-            null as unknown as string,
+            '123' as any as number,
+            TEST_AUTHORIZATION[1],
             TEST_AUTHORIZATION[2],
           ],
         }),
-      ).toThrow('Missing contractAddress parameter');
+      ).toThrow(
+        'Invalid chainId: must be a non-negative number less than 2^256',
+      );
+    });
+
+    it('should throw if chainId is too large', () => {
+      expect(() =>
+        signEIP7702Authorization({
+          privateKey: TEST_PRIVATE_KEY,
+          authorization: [
+            2 ** 257,
```

Comment:
Very nit: We could have used `2 ** 256` since that's the upper bound for that range too (which would have "strictly" matched this `assert auth.chain_id < 2**256` from the EIP)